src/date.js

```diff
@@ -63,3 +63,14 @@ export function withinDates (min, max, date) {
   const mDate = moment(date)
   return mDate.isBetween(min, max, 'day') || mDate.isSame(min, 'day') || mDate.isSame(max, 'day')
 }
+
+export function setDateToZeroTime (date) {
```

Comment:
This mutates the original value; make sure it gets cloned first – and add test for that. !fix